docs/community.md

```diff
@@ -1,11 +1,15 @@
 ## Community Meetings
--   [GitHub discussion regarding Community Meetings](https://github.com/Checkmarx/kics/discussions/5675).
-    -   Here you have access to the PDfs that contains the topics mentioned in each session.
-- Calendar for the upcoming sessions:
-    - October 6, 2022 at 15h00 UTC +0
-    - October 20, 2022 at 15h00 UTC +0
-    - November 3, 2022 at 15h00 UTC +0
-- [To participate in our community meetings you can use this link](https://tinyurl.com/kics-community)
+Our community meetings are held virtually on the first Thursday of every month at 15:00 UTC(+0) by [Zoom](https://tinyurl.com/kics-community).
+
+Upcoming meetings:
+- October 6th, 2022
```

Comment:
```suggestion
```
Since these changes will be released and available to the users only in the next week, this meeting date will be no longer valid.